docs/topics/multiplatform/multiplatform-publish-lib.md

```diff
@@ -156,6 +156,48 @@ kotlin {
 ```
 
 > It is not recommended that you publish variants grouped by the product flavor in case they have different dependencies, 
-> as those will be merged into one dependencies list.
+> as those will be merged into one dependency list.
 >
-{type="note"}
\ No newline at end of file
+{type="note"}
+
+## Disable sources publication
+
+By default, the Kotlin Multiplatform Gradle plugin publishes sources for all the specified targets. However,
+you can configure and disable sources publication with the `withSourcesJar()` API:
+
+* To disable sources publication for all the targets:
+
+  ```kotlin
+  kotlin {
+      withSourcesJar(publish = false)
+  
+      jvm()
+      ios()
```

Comment:
Maybe for future us, we should no longer use shortcuts for targets like `ios()`. We are going to deprecate them in 1.9.20. But it up to you.

Reply:
Thank you! I agree it makes sense not to use them in new texts, plus we can align these three examples